### DISCUSSION TITLE
storybook - module not fount 에러

### DISCUSSION THREAD

**Discussion by Chyyni** [4-2-mogazoa/mogazoa#23] [트러블슈팅]
## 문제 상황
Count라는 컴포넌트 작성 후, story를 만들었는데, storybook을 실행시키니
`Module not found: Error: Can't resolve '@/components/common/Count'`
`Module not found: Error: Can't resolve '@/utils/utils'` 와 같은 에러가 발생했다.

## 해결 방법
import 문에 사용한 `@`를 상대경로(../../)로 바꿔주었더니 해결됐다.

## 이유는?
이유가 뭘까요.. 근본적인 해결 방법을 잘 모르겠습니다. 그렇다고 storybook 때문에 @를 사용하지 않을수도 없고..

**Comment by ckswnskfk** [comment]
저도 이유는 모르고 그냥 그렇다고만 어디서 봤네요..
storybook에서만 상대 경로 쓰는 게 어떨까요?

**Comment by rladngus133** [comment]
```ts
import path from "path";
import type { StorybookConfig } from "@storybook/nextjs";
const config: StorybookConfig = {
   .....<config>>>,
  webpackFinal: async (config: any) => {
    // Add path aliases
    config.resolve.alias["@"] = path.resolve(__dirname, "../src");
    config.resolve.alias["@/pages"] = path.resolve(__dirname, "../src/pages");
    config.resolve.alias["@/components"] = path.resolve(
      __dirname,
      "../src/components"
    );

    return config;
  },
};


export default config;
```

이러한 방법으로 해결했다고 하네요 :) 

스토리북에서 제공하는 공식문서 첨부해드리겠습니다.
공식문서 ) https://storybook.js.org/docs/builders/webpack
참조 ) https://github.com/storybookjs/storybook/issues/12449
